LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Inpsyde GmbH
+Copyright (c) Syde GmbH
```

Comment:
In the GitHub template for MIT license the years is mandatory. I'm gine removing it if we're sure it is not needed.